contribution/gql_mutations.py

```diff
@@ -50,10 +50,10 @@ def reset_premium_before_update(premium):
 
 
 def update_or_create_premium(data, user):
-    uuid = data.pop('uuid', None)
-    if uuid:
+    premium_uuid = data.pop('uuid', None)
+    if premium_uuid:
```

Comment:
@hirensoni913 What does it change exactly? I think both versions would work the same in python?